app/components/forms/wizard/basic-details-step.js

```diff
@@ -497,4 +496,4 @@ export default Component.extend(FormMixin, EventWizardMixin, {
       this.onValid(() => {});
     }
   }
-});
+});
```

Comment:
Codacy found an issue: [Newline required at end of file but not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6140894)